packages/eslint-config-nimble/lib/index.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     './rules/typescript',
     './rules/import',
     './rules/testing',
+    './rules/prettier',
```

Comment:
There used to be a comment `// prettier should be the last one`. Was it important to follow it? If yes, when generating a new project with the React-variant of this template, does it have prettier as the last item of `extends`? 🤔

Reply:
@malparty Good catch, I haven't tried, but I don't have any ideas to try. Should I bring back `prettier` to React package, then it can override rules again on React package 😄

Reply:
@carryall any idea of the side effects? 🤔 
@hoangmirs yeah, having it in both places would be a workaround if we can't figure it out :D

Reply:
Not so sure about the effect but if it's not the last its rules can be overridden by the followed rules, so agree on having it on both packages 🙇🏼